### ISSUE TITLE
Symbol already defined

### ISSUE THREAD

**Issue by Liam0205** [jjolano/shadow#92]
I noticed that version 3.6.0 has been released these days. However when upgrading to 3.6.0 the package manager reports that the Shadow class and the ShadowService class are defined multiple times. When the program is running, one of them will be used but which one is undefined. Specifically, it depends on the order of loading dynamic libraries. I believe this might be a problem, and might cause unstable. Please kindly have a check on it and fix it.

![44E69A52-04B0-4A13-9074-D5F11C242C12](https://user-images.githubusercontent.com/3348485/214258191-60038843-2b6c-4827-a518-686d9ed3a3c3.png)

**Comment by jjolano** [comment]
I am aware of this. This is due to the injection changes, but in terms of stability there should be nothing to worry about. I will try to suppress the message in an update.

**Comment by jjolano** [comment]
Can you confirm if this build fixes the issue?

[me.jjolano.shadow_3.6.0-3+debug_iphoneos-arm.zip](https://github.com/jjolano/shadow/files/10494596/me.jjolano.shadow_3.6.0-3%2Bdebug_iphoneos-arm.zip)

**Comment by Liam0205** [comment]
> Can you confirm if this build fixes the issue?
> 
> [me.jjolano.shadow_3.6.0-3+debug_iphoneos-arm.zip](https://github.com/jjolano/shadow/files/10494596/me.jjolano.shadow_3.6.0-3%2Bdebug_iphoneos-arm.zip)

```
bash-5.0# dpkg -i "/var/mobile/Library/Mobile Documents/com~apple~CloudDocs/Downloads/me.jjolano.shadow_3.6.0-3+debug_iphoneos-arm.deb" ;
(Reading database ... 
(Reading database ... 5%
(Reading database ... 10%
(Reading database ... 15%
(Reading database ... 20%
(Reading database ... 25%
(Reading database ... 30%
(Reading database ... 35%
(Reading database ... 40%
(Reading database ... 45%
(Reading database ... 50%
(Reading database ... 55%
(Reading database ... 60%
(Reading database ... 65%
(Reading database ... 70%
(Reading database ... 75%
(Reading database ... 80%
(Reading database ... 85%
(Reading database ... 90%
(Reading database ... 95%
(Reading database ... 100%
(Reading database ... 7377 files and directories currently installed.)
Preparing to unpack .../me.jjolano.shadow_3.6.0-3+debug_iphoneos-arm.deb ...
Unpacking me.jjolano.shadow (3.6.0-3+debug) over (3.5.6) ...
Setting up me.jjolano.shadow (3.6.0-3+debug) ...
shdw: successfully regenerated dpkg ruleset
```

**Comment by jjolano** [comment]
Should be fixed in 3.6.1.

**Comment by Liam0205** [comment]
> Should be fixed in 3.6.1.

Glad to know! 😋
